src/main/java/ticTacToe/game/GameFlow.java

```diff
@@ -80,8 +68,7 @@ private void reportFinalResult() {
     }
 
     private void startNewGameOrQuit() {
-        String answer = ui.askToPlayAgain();
-        if (answer.equals("y")) {
+        if (ui.askToPlayAgain().equals("y")) {
```

Comment:
Minor point, it might read nicer if you have something like `if (playAgain(userInput))`